### PR TITLE
Style guideline fixes, screenshot additions

### PR DESCRIPTION
**Pull request by Jessica-c53**
I am new at editing, so there may be errors in attaching images to the documentation text edits!

### CODE REVIEW

**Comment by amyblais** [comment]
Hey @Jessica-c53! Tips for adding screenshots:

1. The image file name must match the name you use in the doc, e.g. "Create private channel.png" file name doesn't match "Create_private_channel.png" in the doc, so it won't render properly. Same with "add memebers.png".
2. There was no reference to "channel options mobil.jpg" image in the doc.

Please make these file adjustments, otherwise the doc edits looked good!

If you have any questions, let me know!

**Comment by Jessica-c53** [comment]
I edited the image names and deleted the extra graphic that I didn't include. Hope all the fixes work!

**Comment by amyblais** [comment]
It looks like spaces between words doesn't work for file names (I wasn't aware of that), so the best option would be to change the actual file names (removing any spaces) and to re-upload them here. (Also noticed that "add memebers.png" has a typo).

**Comment by amyblais** [comment]
The images are still not rendering properly after you fixed the file names, and I noticed it's because they should be uploaded in docs > source > images (https://github.com/mattermost/docs/tree/master/source/images) instead of the main docs page.

I'm so sorry you have to re-upload the screenshots again :(

**Comment by Jessica-c53** [comment]
No worries! Learning process for me :). Hope that was the final issue!

**Comment by amyblais** [comment]
Awesome! Just fixed a few formatting issues. Giving this for a PM to review the screenshots.

File: source/help/getting-started/organizing-conversations.rst [hunk 1]
```diff
@@ -1,57 +1,79 @@
...
 
 Public Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Public Channels are open to everyone on a team. New team members are automatically added to two Public Channels when they sign up: Town Square and Off-Topic.
 
 Private Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Private Channels are for sensitive topics and are only visible to select team members. Any member of a Private Channel can add additional members. Channel members can choose to leave at any time, but only the channel owner or `Team Admin <http://docs.mattermost.com/help/getting-started/managing-members.html#user-roles>`_ can remove other members.
```
Comment (lfbrock) [inline source/help/getting-started/organizing-conversations.rst]: @amyblais - would you be able to double check if this is correct for Team Edition as well? I think anyone may be able to remove people from channels in Team Edition, and in Enterprise Edition you can change the permissions to be Channel Admin and System Admin.

Comment (amyblais) [reply source/help/getting-started/organizing-conversations.rst]: @esethna Would you be able to help confirm that ^

Comment (esethna) [reply source/help/getting-started/organizing-conversations.rst]: I believe LB is correct. Team Edition any channel members can add or remove members in public and private channels

**Pull request merged**